wails-frontend/src/pages/failure.tsx

```diff
@@ -0,0 +1,38 @@
+import { useLocation } from 'react-router';
+import { promptAction, promptRequest } from '../events/events';
+import { FiX } from 'react-icons/fi';
+
+const Failure = () => {
+  const { state } = useLocation();
+  const { req } = state;
+
+  const successMsg = (req: promptRequest) => {
+    switch (req.Action) {
+      case promptAction.deleteReq:
+        return 'The account could not been deleted';
+      case promptAction.newPasswordReq:
+        return 'The password could not be created';
+      case promptAction.importReq:
+        return 'The account could not be imported';
+      case promptAction.signReq:
+        return 'The transaction has been signed';
+      default:
+        return 'Apply';
```

Comment:
Is that correct have a default with Apply!? 🤔

Reply:
no not correct will make changes

Reply:
chenged !